getting-started/Frontend/emojify/src/main/java/com/google/cloud/kotlin/emojify/Application.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.kotlin.emojify
+
+import com.yanzhenjie.album.Album
+import com.yanzhenjie.album.AlbumConfig
+import java.util.Locale
+
+class Application : android.app.Application() {
+
+    companion object {
+        var instance: Application? = null
```

Comment:
do you really need this?

Reply:
If you do need this I would recommend making it a lateinit var with a private setter. Only ContentProviders can exist without your Application's onCreate being called first. These two things will remove null checks and prevent some rando case from setting it.